cmd/avrogo/generate.go

```diff
@@ -42,7 +42,13 @@ func generate(w io.Writer, pkg string, ns *parser.Namespace, definitions []schem
 		imports:  make(map[string]string),
 		extTypes: extTypes,
 	}
-	gc.addImport("github.com/heetch/avro/avrotypegen")
+	// Add avrotypegen package conditionally when there is a RecordDefinition in it.
+	for _, def := range ns.Definitions {
+		if typeof(def) == "RecordDefinition" {
```

Comment:
Why not use type assertion here, with something like
```
if _, ok := def.(*schema.RecordDefinition); ok {
```

This checks against the real type instead of comparing to a string?

Reply:
Done in adb77d0